### PR TITLE
Update kitchen yaml to be toolchain aware, create fakeroot local def

### PR DESCRIPTION
**Pull request by scotthain**

### CODE REVIEW

**Comment by scotthain** [comment]
This will be updated to remove the fakeroot def and use it after https://github.com/chef/omnibus-software/pull/683 is merged.

@chef/engineering-services 

This will resolve https://github.com/chef-cookbooks/omnibus/pull/173 after a release and version bump.

File: config/software/omnibus-toolchain.rb [hunk 1]
```diff
@@ -28,6 +28,9 @@
 # git!
 dependency "git"
 
+# fakeroot for EL based things
+dependency "fakeroot" if el?
```
Comment (schisamo) [inline config/software/omnibus-toolchain.rb]: Is there any reason we wouldn't want this available in our toolchain for all platforms?

Comment (scotthain) [inline config/software/omnibus-toolchain.rb]: I don't know of any, I was following the example of the omnibus cookbook. I thought it was only used during RPM creation? I'm fine with removing this condition if we want to go that direction.

Comment (schisamo) [inline config/software/omnibus-toolchain.rb]: Yeah the only reason that conditional existed was the `fakeroot` package only existed for EL. Now that we are compiling from source we should make it available to all platforms.

**Comment by jaymalasinha** [comment]
👍

**Comment by schisamo** [comment]
👍

**Pull request merged**